content/may2015/authorization-specification.md

```diff
@@ -0,0 +1,1029 @@
+---
```

Comment:
I assume the same comments apply to the may2015 versions of the files as the dstu2 versions